components/src/atoms/radio/Radio.js

```diff
@@ -19,24 +23,24 @@ export default {
       required: true
     },
     /**
-     * Legend text
+     * Input name
      */
-    legend: {
+    id: {
       type: String,
-      default: null
+      required: true
     },
     /**
-     * Class for input
+     * Input value
      */
-    inputClass: {
-      type: String,
-      default: null
+    value: {
+      type: [String, Number, Boolean],
+      required: true
```

Comment:
I think in some cases value will be the same as `id`, maybe let's change `value` to not be required and use `id` in this case. 
Then instead of
```
<a-radio
  v-model="selected"
  name="radio1"
  id="option_one"
  value="option_one"
>
  Option one
</a-radio>
```
we can use
```
<a-radio
  v-model="selected"
  name="radio1"
  id="option_one"
>
  Option one
</a-radio>
```
What do you think?

Reply:
added computed inputId 
```
  computed: {
    inputId () {
      return this.id || this.value.toLowerCase().replace(/[^a-z0-9]/g, '-')
    }
  }
```